[Response Ops][Cases] Use deprecation object (#201004)
elastic/kibana: docs/upgrade-notes.asciidoc

[discrete]

> [[breaking-201004]]

Reviewer: How did you find out these needed to be updated too?

Author: Because [the link](https://github.com/elastic/kibana/pull/201004/files/3af43a82c998b4d608685072811e838bec54ee19#diff-d7680a659a0b45b0ec187364d27830ef8d2463becdbd40ca05abf1f44aff0480R31) to the docs' reference is [this value](https://github.com/elastic/kibana/pull/201004/files/3af43a82c998b4d608685072811e838bec54ee19#diff-d7680a659a0b45b0ec187364d27830ef8d2463becdbd40ca05abf1f44aff0480R8). I just kept "breaking" because of consistency with the other references

[discrete]
[[breaking-201004]]
.[Cases] Legacy deprecations (9.0.0)
[%collapsible]
====
*Details* +
`GET /api/cases/status` has been deprecated with no replacement. Deleted in v9.0.0

`GET /api/cases/{case_id}/comments` has been replaced by `GET /api/cases/{case_id}/comments/_find` released in v7.13

`GET /api/cases/<case_id>/user_actions` has been replaced by `GET /api/cases/<case_id>/user_actions/_find` released in v8.7

> `includeComments` parameter in `GET /api/cases/{case_id}`has been deprecated. Use `GET /api/cases/{case_id}/comments/_find` instead, released in v7.13

Reviewer 2: ```suggestion
`includeComments` parameter in `GET /api/cases/{case_id}` has been deprecated. Use `GET /api/cases/{case_id}/comments/_find` instead, released in v7.13
```

*Impact* +
Deprecated endpoints will fail with a 404 status code starting from version 9.0.0

*Action* +
Remove references to `GET /api/cases/status` endpoint.

> Replace references to endpoints listed as deprecated by it's replacement. See `Details` section 

Reviewer 2: ```suggestion
Replace references to deprecated endpoints with the replacements listed in the breaking change details.
```
